packages/app/src/vis-packs/core/compound/MappedCompoundVis.tsx

```diff
@@ -49,12 +53,19 @@ function MappedCompoundVis(props: Props) {
     slicedMapping,
   );
 
+  const selection = getSliceSelection(dimMapping);
   const fieldFormatters = Object.values(fields).map((field) =>
     getFormatter(field, notation),
   );
 
-  const { getExportURL } = useDataContext();
-  const selection = getSliceSelection(dimMapping);
+  const exportEntries = useExportEntries(['npy', 'csv'], dataset, selection, {
+    csv: () =>
+      generateCsv(
+        fieldNames,
+        mappedArray,
+        Object.values(fields).map((field) => getCsvFormatter(field)),
+      ),
```

Comment:
Each mapped component that supports CSV exports implements its own CSV exporter: `MappedLineVis`, `MappedCompoundVis` and `MappedMatrixVis`.